sos/collector/transports/__init__.py

```diff
@@ -303,7 +303,20 @@ def retrieve_file(self, fname, dest):
         :returns:   True if file was successfully copied from remote, or False
         :rtype:     ``bool``
         """
-        return self._retrieve_file(fname, dest)
+        attempts = 0
+        try:
+            while attempts < 5:
+                attempts += 1
+                ret = self._retrieve_file(fname, dest)
+                if ret:
+                    return True
+                self.log_info("File retrieval attempt %s failed" % attempts)
+            self.log_info("File retrieval failed after 5 attempts")
```

Comment:
maybe set it to `log_error` so that it's visible that file wasn't downloaded correctly?

Reply:
Will be visible via https://github.com/sosreport/sos/pull/2804/files#diff-c64b04e837e3b06a12632c6b923d50b18f88884c482c71973d43a6738b49f3dcR807

Reply:
Ah, right, I thought at first that `ui_msg` has lower visibility than `log_err`
But it doesn't really say ERROR, so log message looks exactly like "Retrieving sos report..." for example, maybe add something like "ERROR:" to the message itself?